sdk/common/src/main/java/io/opentelemetry/sdk/common/InstrumentationLibraryInfo.java

```diff
@@ -1,72 +1,71 @@
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.opentelemetry.sdk.common;
-
-import com.google.auto.value.AutoValue;
-import io.opentelemetry.sdk.trace.TracerSdkProvider;
-import io.opentelemetry.trace.Tracer;
-import javax.annotation.Nullable;
-import javax.annotation.concurrent.Immutable;
-
-/**
- * Holds information about the instrumentation library specified when creating an instance of {@link
- * Tracer} using {@link TracerSdkProvider}.
- */
-@AutoValue
-@Immutable
-public abstract class InstrumentationLibraryInfo {
-  private static final InstrumentationLibraryInfo EMPTY = create("", null);
-
-  /**
-   * Creates a new instance of {@link InstrumentationLibraryInfo}.
-   *
-   * @param name name of the instrumentation library (e.g., "io.opentelemetry.contrib.mongodb"),
-   *     must not be null
-   * @param version version of the instrumentation library (e.g., "semver:1.0.0"), might be null
-   * @return the new instance
-   */
-  public static InstrumentationLibraryInfo create(String name, @Nullable String version) {
-    return new AutoValue_InstrumentationLibraryInfo(name, version);
-  }
-
-  /**
-   * Returns an "empty" {@code InstrumentationLibraryInfo}.
-   *
-   * @return an "empty" {@code InstrumentationLibraryInfo}.
-   */
-  public static InstrumentationLibraryInfo getEmpty() {
-    return EMPTY;
-  }
-
-  /**
-   * Returns the name of the instrumentation library.
-   *
-   * @return the name of the instrumentation library.
-   */
-  public abstract String getName();
-
-  /**
-   * Returns the version of the instrumentation library, or {@code null} if not available.
-   *
-   * @return the version of the instrumentation library, or {@code null} if not available.
-   */
-  @Nullable
-  public abstract String getVersion();
-
-  // Package protected ctor to avoid others to extend this class.
-  InstrumentationLibraryInfo() {}
-}
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.common;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.trace.Tracer;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Holds information about the instrumentation library specified when creating an instance of {@link
+ * Tracer} using io.opentelemetry.sdk.trace.TracerSdkProvider.
```

Comment:
for some unknown reason this line no longer uses the imported class, but the fully-qualified one. I think it'll make the diff slightly cleaner if that change got reverted.

Reply:
I had to do that because the class is in the "tracing" module, which isn't a dependency of "common".

As it was for Javadoc purposes, I thought it would be ok.

Are there alternatives? I couldn't find an easy way to reference a class in another module without adding a bunch of things to the javadoc command line

Reply:
hmm. good point. I might get rid of the reference altogether to the concrete class, and just reference something like "Tracer or Meter Provider".

Reply:
Will do